docs/getting-started.md

```
@@ -0,0 +1,184 @@
+
+### 3. 依存関係のインストール
+
+```bash
+bun install
+```
+
+このコマンドは以下を実行します：
+
+- npm パッケージのインストール
+- サブモジュールの初期化（`preinstall` スクリプト）
+- Git フック（lefthook）のセットアップ（`prepare` スクリプト）
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)